cmd/skaffold/app/cmd/init.go

```diff
@@ -42,7 +42,7 @@ func NewCmdInit() *cobra.Command {
 			f.BoolVar(&skipBuild, "skip-build", false, "Skip generating build artifacts in Skaffold config")
 			f.BoolVar(&force, "force", false, "Force the generation of the Skaffold config")
 			f.StringVar(&composeFile, "compose-file", "", "Initialize from a docker-compose file")
-			f.StringSliceVarP(&cliArtifacts, "artifact", "a", nil, "'='-delimited dockerfile/image pair to generate build artifact\n(example: --artifact=/web/Dockerfile.web=gcr.io/web-project/image)")
+			f.StringArrayVarP(&cliArtifacts, "artifact", "a", nil, "'='-delimited builder JSON/image pair to generate build artifact\n(example: --artifact='{\"name\":\"Docker\",\"payload\":{\"path\":\"/web/Dockerfile.web\"}}=gcr.io/web-project/image')")
```

Comment:
Just asking... won't `/web/Dockerfile.web` read more natural without the leading `/`? It may usually be a relative path? I can't remember exactly, but can this handle absolute path properly?